### ISSUE TITLE
Keyboard Navigation, using arrows to navigate doesn't allow slider movement

### ISSUE THREAD

**Issue by CWilson4960** [phetsims/resistance-in-a-wire#115]
Test device: Dell xps (Hanson)

Operating System: Windows 10 1703.15063.673

Browser: Mozilla Firefox 56.0

Problem description: You are no longer able to adjust the slider with arrows after using arrow keys to navigate to the sliders, as opposed to using tab.

Steps to reproduce:
1) Load sim
2) Tab through the controls until the focus box is on the reset button.
3) Use the arrow keys to navigate the focus box up to the Area slider, or length slider, or resistivity one
4) Try to adjust sliders using arrow keys
phetsims/QA/issues/66
URL: https://www.colorado.edu/physics/phet/dev/html/resistance-in-a-wire/1.4.0-rc.1/resistance-in-a-wire_en.html
Version: 1.4.0-rc.1 2017-11-09 21:20:24 UTC
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:56.0) Gecko/20100101 Firefox/56.0
Language: en-US
Window: 1280x612
Pixel Ratio: 2.5/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0
Vendor: Mozilla (Mozilla)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {"assert":{"sha":"928741cf","branch":"HEAD"},"axon":{"sha":"b8dd4e51","branch":"HEAD"},"babel":{"sha":"db7312fb","branch":"master"},"brand":{"sha":"cfca902d","branch":"HEAD"},"chipper":{"sha":"b8a3c09a","branch":"HEAD"},"dot":{"sha":"84bc5146","branch":"HEAD"},"joist":{"sha":"3eff115f","branch":"HEAD"},"kite":{"sha":"91e2ccf1","branch":"HEAD"},"phet-core":{"sha":"5b0df3e8","branch":"HEAD"},"phetcommon":{"sha":"5b0c8dd6","branch":"HEAD"},"query-string-machine":{"sha":"c74e454e","branch":"HEAD"},"resistance-in-a-wire":{"sha":"e164e33b","branch":"HEAD"},"scenery":{"sha":"db7ea875","branch":"HEAD"},"scenery-phet":{"sha":"71f32bfc","branch":"HEAD"},"sherpa":{"sha":"0d590744","branch":"HEAD"},"sun":{"sha":"79c0dece","branch":"HEAD"},"tandem":{"sha":"513bc954","branch":"HEAD"}}

**Comment by jessegreenberg** [comment]
Thanks @CWilson4960, this is weird.  I am unable to reproduce with Windows 10 Firefox 56.0.2. 

> Use the arrow keys to navigate the focus box up to the Area slider, or length slider, or resistivity one

How are you able to do this? Do you have custom settings to enable navigation with arrow keys? Or were you by chance using a screen reader prior to testing?

**Comment by jessegreenberg** [comment]
I just tried to reproduce with Hanson testing machine, when I turned volume on NVDA started talking to me, so the screen reader was enabled. When I turn the screen reader off,  the problem goes away. I can no longer use the arrow keys to move focus. Closing.

**Comment by CWilson4960** [comment]
Thank you Jesse! Sorry I didn't catch that! I hope it wasn't too much of an inconvenience.

**Comment by jessegreenberg** [comment]
Not at all, it was definitely more of an inconvenience for you, I was most likely the last one who left it on. Sorry about that!
